distribution/docs/src/main/resources/managing.adoc

```diff
@@ -87,9 +87,15 @@ include::${project.build.directory}/doc-contents/_contents/_nsili/managing-nsili
 <<<
 
 == Managing ${alliance-video}
-[small]#Version: ${ddf.version}#
+[small]#Version: ${project.version}#
```

Comment:
Should we have an `alliance.version` param for this substitution or does this grab maven properties?